src/Components/Patient/FileUpload.tsx

```diff
@@ -1294,20 +1293,15 @@ export const FileUpload = (props: FileUploadProps) => {
                 <>
                   <VoiceRecorder createAudioBlob={createAudioBlob} />
                   {audioBlob && (
-                    <Button
-                      color="primary"
-                      variant="contained"
-                      type="submit"
-                      style={{ marginLeft: "auto" }}
-                      startIcon={
-                        <CloudUploadOutlineIcon>save</CloudUploadOutlineIcon>
-                      }
+                    <ButtonV2
+                      className="btn btn-primary flex gap-2"
```

Comment:
@Pranshu1902 when using ButtonV2 use variants and other props to make styles. avoid using classNames unless you need to break out of the standard designs.
```suggestion
```